ui/src/widgets/ui-gauge/UIGauge.vue

```diff
@@ -41,7 +41,7 @@
             </g>
         </svg>
         <div ref="value" class="nrdb-ui-gauge-value" :class="'nrdb-ui-' + props.gtype">
-            <span>{{ props.prefix }}{{ value || props.min }}{{ props.suffix }}</span>
+            <span>{{ props.prefix }}{{ value !== undefined ? value : props.min }}{{ props.suffix }}</span>
```

Comment:
Joe, 

it looks like you are handling `0` here right?

I think this should use [nullish coalescing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing) here.  What if the value is `null`?

```suggestion
            <span>{{ props.prefix }}{{ value ?? props.min }}{{ props.suffix }}</span>
```

* if the value is `0`, then `0` will be used
* if the value is `undefined`, then `props.min` will be used
* if the value is `null`, then `props.min` will be used
* if the value is `-1`, then `-1` will be used

However, if there is other code that absolutely prevents `null` making its way to this point, then your solution is fine too.